libsplinter/src/oauth/store/mod.rs

```diff
@@ -0,0 +1,19 @@
+// Copyright 2018-2020 Cargill Incorporated
```

Comment:
We're planning on having more than one store that belongs to OAuth, right? Probably want this module to have a more specific name.

Reply:
We can rearrange them when we create the next one

Reply:
Or, there should be one OAuth store